### PR TITLE
OSF Changes

### PR DESCRIPTION
**Pull request by alexandrurosu29**
Added changes according to the new specifications.

### CODE REVIEW

File: src/main/java/com/constantcontact/ConstantContactFactory.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+import com.constantcontact.services.emailcampaigns.schedule.EmailCampaignScheduleService;
+import com.constantcontact.services.emailcampaigns.schedule.IEmailCampaignScheduleService;
+import com.constantcontact.services.emailcampaigns.tracking.EmailCampaignTrackingService;
+import com.constantcontact.services.emailcampaigns.tracking.IEmailCampaignTrackingService;
+import com.constantcontact.services.eventspot.EventSpotService;
+import com.constantcontact.services.eventspot.IEventSpotService;
+import com.constantcontact.services.library.IMyLibraryService;
+import com.constantcontact.services.library.MyLibraryService;
+
+public class ConstantContactFactory {
```
Comment (csciuto) [inline src/main/java/com/constantcontact/ConstantContactFactory.java]: Could you please provide JavaDoc?

**Comment by csciuto** [comment]
Looks good! I had one comment about the new ConstantContactFactory class having no JavaDoc. Other than that, could you please update pom.xml to point to the next major version (which I believe is 3.0.0)?

Thanks.

**Comment by halusstefan** [comment]
Hi @csciuto ,

Please accept this pull request as it is, so I can pull these changes into my own fork and make the version change and to fill java docs for ConstantContactFactory class.

Thanks,
Stefan.

**Comment by csciuto** [comment]
Talkd to @halusstefan , doing as requested.

**Pull request merged**